File: exercicio5.py
nota1 = int(input("Digite a primeira nota: "))
nota2 = int(input("Digite a segunda nota: "))
nota3 = int(input("Digite a terceira nota: "))

# notas = [nota1, nota2, nota3] # versão que eu adotaria no dia a dia
# media = sum(notas) / len(notas) # versão que eu adotaria no dia a dia

notas = nota1 + nota2 + nota3 # versão mais coerente com a proposta do material
media = notas / 3 # versão mais coerente com a proposta do material

if media >= 7:
    print(f"Média: {media:.2f}. Aluno aprovado.")
else:
    print(f"Média: {media:.2f}. Aluno não aprovado.")

File: 2023/ATIVIDADEI/CPP/exercicio5.cpp
#include <iostream>

using namespace std;

int main() {
    int soma, media, comprimento, nota1, nota2, nota3;

    cout << "Digite a primeira nota: " << endl;
    cin >> nota1;
    cout << "Digite a segunda nota: " << endl;
    cin >> nota2;
    cout << "Digite a terceira nota: " << endl;
    cin >> nota3;

    int lista[] = {nota1, nota2, nota3};

    soma = nota1 + nota2 + nota3;
    comprimento = sizeof(lista) / sizeof(lista[0]);

    cout << sizeof(lista) << endl;
    cout << sizeof(lista[0]) << endl;
    cout << comprimento << endl ;

    media = soma / comprimento;
    cout << "A média de notas deste aluno é: " << media << endl;

    if (media >= 7) {
        cout << "Aluno Aprovado!" << endl;
    } else {
        cout << "Aluno não foi aprovado" << endl;
    };

    return 0;
}